src/InlineAutocomplete/InlineAutocomplete.features.stories.tsx

```diff
@@ -1,10 +1,10 @@
 import React, {useState} from 'react'
 import {Meta} from '@storybook/react'
-import {ActionList, Avatar, ActionListItemProps, Textarea, TextInput, FormControl} from '../..'
+import {ActionList, Avatar, ActionListItemProps, Textarea, TextInput, FormControl} from '..'
 import InlineAutocomplete, {ShowSuggestionsEvent, Suggestions} from '.'
 
 export default {
-  title: 'Components/Forms/InlineAutocomplete/Features',
+  title: 'Components/InlineAutocomplete/Features',
```

Comment:
Sorry, I didn't realised this component is draft. For draft components, we should serve them under the Drafts folder in storybook. 
```suggestion
  title: 'Drafts/Components/InlineAutocomplete/Features',
```